Examples/Examples/ContentView.swift

```diff
@@ -47,6 +50,24 @@ struct ContentView: View {
                         Label("Load HTML String", systemImage: "doc")
                             .labelStyle(.iconOnly)
                     }
+
+                    Button {
+                        Task {
+                            pdf = PDFDocument(data: try! await proxy.contentReader.pdf())
+                        }
+                    } label: {
+                        Label("Take PDF", systemImage: "printer")
+                            .labelStyle(.iconOnly)
+                    }
+                    .accessibilityIdentifier("take_pdf_button")
+
+                    if let pdf {
+                        ShareLink(item: pdf,
+                                  preview: SharePreview("PDF Document",
+                                                        image: Image(uiImage: (pdf.page(at: 0)?.thumbnail(of: CGSize(width: 100, height: 100),
+                                                                                                          for: .cropBox))!)))
+                        .labelStyle(.iconOnly)
+                    }
```

Comment:
```suggestion
                    if let pdf, let thumbnail = pdf.thumbnail {
                        ShareLink(item: pdf, preview: SharePreview("PDF Document", image: thumbnail))
                            .labelStyle(.iconOnly)
                    }
```

Additionally, the following extension must be implemented:

```swift
private extension PDFDocument {
    var thumbnail: Image? {
        guard let firstPage = page(at: 0) else { return nil }
        return Image(uiImage: firstPage.thumbnail(of: CGSize(width: 100, height: 100), for: .cropBox))
    }
}
```

Reply:
fixed in [f993775](https://github.com/cybozu/WebUI/pull/58/commits/f9937754e077afb063d71734dd4277b644b278e7)